sql/core/src/test/resources/tpcds-query-results/v1_4/q78.sql.out

```diff
@@ -0,0 +1,6 @@
+-- Automatically generated by TPCDSQueryTestSuite
+
+-- !query schema
+struct<ratio:double,store_qty:bigint,store_wholesale_cost:decimal(17,2),store_sales_price:decimal(17,2),other_chan_qty:bigint,other_chan_wholesale_cost:decimal(18,2),other_chan_sales_price:decimal(18,2)>
+-- !query output
+
```

Comment:
This query returns nothing when `sf=1`. I've checked this query returns output rows if `sf` increases, but I think larger data sets can make a hight pressure on GA jobs, so I keep this test as it is just for output schema checks.